sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorTraceExporter.java

```diff
@@ -254,22 +253,22 @@ private void exportRemoteDependency(SpanData span, boolean inProc,
     }
 
     private void applySemanticConventions(Attributes attributes, RemoteDependencyData remoteDependencyData, SpanKind spanKind) {
-        String httpMethod = attributes.get(SemanticAttributes.HTTP_METHOD);
+        String httpMethod = attributes.get(AttributeKey.stringKey("http.method"));
         if (httpMethod != null) {
             applyHttpClientSpan(attributes, remoteDependencyData);
             return;
         }
-        String rpcSystem = attributes.get(SemanticAttributes.RPC_SYSTEM);
```

Comment:
Were these constants removed from SemanticAttributes in the GA release?

Reply:
So we removed the dependency on semconv package in this PR https://github.com/Azure/azure-sdk-for-java/pull/18940 since it was moved out of the core oTel lib and was to be managed independently (possible breaking changes). I missed doing these updates to the exporter lib in the last PR so covering them now.